src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/json/StatementOfCapitalDataJson.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.confirmationstatementapi.model.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.confirmationstatementapi.model.StatementOfCapital;
+import uk.gov.companieshouse.confirmationstatementapi.model.tasks.TaskStatus;
+
+public class StatementOfCapitalDataJson {
+
+    @JsonProperty("task_status")
+    private TaskStatus taskStatus;
+
+    @JsonProperty("statement_of_capital")
+    private StatementOfCapital statementOfCapital;
```

Comment:
The StatementOfCapital looks like it is in the wrong package, seems to be a json model similar to this one. I would move into this package and add Json to the class name to make it consistent with the rest of the model.

Reply:
You just said don't do that!